uPortal-groups/uPortal-groups-core/src/main/java/org/apereo/portal/groups/RDBMEntityGroupStore.java

```diff
@@ -86,7 +86,7 @@
     private static String insertMemberSql;
 
     // SQL group search string
-    private static String searchGroupsPartial =
+    private static String searchGroupsPartialCaseInsensitive =
```

Comment:
These look as though they should be both `final` and `IN_ALL_UPPERCASE`.  I see you were following what was there; but since we're touching it, we should improve it in this no-brainer way (barring some issue I can't see).